operating_unit/models/operating_unit.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
+# © 2015 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class OperatingUnit(models.Model):
+
+    _name = 'operating.unit'
+    _description = 'Operating Unit'
+
+    name = fields.Char('Name', required=True)
```

Comment:
If you field name is equal to you string field you could remove the fiel string.
Odoo use the field name by default as string.

`s/'Name',//g`